prow/config/jobs/istio.yaml

```diff
@@ -238,6 +238,23 @@ jobs:
     - prow/config/ambient-sc.yaml
     - test.integration.ambient.kube
 
+  - name: integ-ambient-ipv6
+    modifiers: [presubmit_optional, presubmit_skipped]
+    requirements: [kind]
+    env:
+    - name: INTEGRATION_TEST_FLAGS
+      value: " --istio.test.kube.helm.values=cni.ambient.ipv6=true --istio.test.ambient "
```

Comment:
ipv6 is default enabled now

Reply:
@bleggett thinks it is better to explicitly enable this, so that people understand the various knobs that are in use, in case the defaults change later on